### PR TITLE
[MKP/doc] Update doc for changes of service account credential.

### PR DESCRIPTION
**Pull request by numerology**
Reflect changes made in #2308

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/pipelines/2309)

### CODE REVIEW

File: manifests/gcp_marketplace/guide.md [hunk 1]
```diff
@@ -17,26 +17,42 @@ Specify a [Kubenetes namespace](https://kubernetes.io/docs/concepts/overview/wor
...
-$ gcloud iam service-accounts keys create application_default_credentials.json --iam-account [your-service-account] && cat application_default_credentials.json | base64
+export CLUSTER=<cluster-where-kfp-was-installed>
+export ZONE=<zone-where-kfp-was-installed>
+# Configure kubectl to connect with the cluster
+gcloud container clusters get-credentials "$CLUSTER" --zone "$ZONE"
 ```
-
-Existing key also can be used.
-
+Then we can create and inject service account credential.
```
Comment (james-jwu) [inline manifests/gcp_marketplace/guide.md]: "Then **you** can create ..."

**Comment by numerology** [comment]
cc @jessiezcc

File: manifests/gcp_marketplace/README.md [hunk 2]
```diff
@@ -134,15 +128,10 @@ export MARKETPLACE_TOOLS_IMAGE=gcr.io/cloud-marketplace-staging/marketplace-k8s-
 Run the install script
 
 ```shell
-mpdev scripts/install  --deployer=gcr.io/ml-pipeline/google/pipelines/deployer:0.2   --parameters='{"name": "'$APP_INSTANCE_NAME'", "namespace": "'$NAMESPACE'", "serviceAccountCredential": "'$SERVICE_ACCOUNT_TOKEN'"}'
+mpdev scripts/install  --deployer=gcr.io/ml-pipeline/google/pipelines/deployer:0.1   --parameters='{"name": "'$APP_INSTANCE_NAME'", "namespace": "'$NAMESPACE'"}'
```
Comment (paveldournov) [inline manifests/gcp_marketplace/README.md]: Is the image tag getting decreased to 0.1 expected?

Comment (numerology) [reply manifests/gcp_marketplace/README.md]: Currently for alpha launch, we'll pin the version to 0.1, so yes.

**Pull request merged**